_data/projects/Circle.yml

```diff
@@ -0,0 +1,12 @@
+name: Circle
+desc: A SVG circle that becomes bigger on button click
+site: https://github.com/Terence-art/circle
+tags:
+  - tags
+  - relevant
+  - to
+  - the
+  - project
+upforgrabs:
+  name: help wanted
+  link: https://github.com/Terence-art/circle/blob/main/circle.html
```

Comment:
This link should go to a list of issues (possibly labeled `help wanted`) instead of a file within the repository.

Any plans to add some issues to the repository? Otherwise this likely won't show up on the site...

Reply:
Is just to make the edit the code because the debug of the local page  doesn't give me errors still it won't work properly.
Because there were no errors I posted for clarify.
Please be patient as is my first pull request